docs/how-tos/provision-setup.md

```diff
@@ -0,0 +1,205 @@
+
+- Navigate to your machine's page on [the Viam app](https://app.viam.com).
+- Select the part status dropdown to the right of your machine's name on the top of the page.
+  {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown">}}
+- Click the copy icon next to **Viam server config** to copy the machine credentials configuration file.
+- Paste the `viam-server` app config when prompted.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**5. Wait for machine to complete setup**
+
+The machine will now disable the hotspot network and attempt to connect using the provided network information.
+If the machine cannot establish a connection using the provided network information, the machine will create the hotspot again and prompt you to re-enter the network information until a connection is successfully established.
+
+If the machine can successfully connect to the network it will now complete its setup and become **live**.
+
+Note that any features that require internet access will not function if the connected WiFi network is not connected to the internet.
+{{% /tablestep %}}
+{{< /table >}}
+
+## Update machine configuration
+
+{{< table >}}
+{{% tablestep link="/configure/agent/#agent-provisioning"%}}
+**1. Update the `agent-provisioning` configuration**
+
+For security purposes, you should change the `hotspot_password` attribute.
+
+Go to your machine's page in [the Viam app](https://app.viam.com) and use the JSON editor.
+
+Look for the `agent` configuration and add line 7 with a suitable password:
+
+```json {class="line-numbers linkable-line-numbers" data-line="7"}
+...
+"agent": {
+  "agent-provisioning": {
+    ...
+    "attributes": {
+      ...
+      "hotspot_password": "NewAndSafePassword",
+    }
+  }
+}
+```
+
+{{% /tablestep %}}
+{{% tablestep link="/configure/agent/#networks"%}}
+**2. Add WiFi networks**
+
+If your machine might move between different WiFi networks, you can enable `roaming_mode` and add additional networks to the [`viam-agent` configuration](/configure/agent/#configuration).
+
+To add additional networks, go to your machine's page in [the Viam app](https://app.viam.com) and use the JSON editor.
+
+Look for the `agent` configuration and add lines 7-19 to your configuration, changing the credentials as to the ones for your networks:
+
+```json {class="line-numbers linkable-line-numbers" data-line="7-20"}
+...
+"agent": {
+  "agent-provisioning": {
+    ...
+    "attributes": {
+      ...
+      "roaming_mode": true,
+      "networks": [
+        {
+          "type": "wifi",
+          "ssid": "fallbackNetOne",
+          "psk": "myFirstPassword",
+          "priority": 30
+        },
+        {
+          "type": "wifi",
+          "ssid": "fallbackNetTwo",
+          "psk": "mySecondPassword",
+          "priority": 10
+        }
+      ]
+    }
+  }
+}
+```
+
+{{% /tablestep %}}
+{{< /table >}}
```

Comment:
I don't think these are needed here. Most users don't need to do these steps. Perhaps just mention them and link the existing section for these configuration options.